libs/platform/src/lib/components/action-bar/action-bar.component.ts

```diff
@@ -48,13 +46,17 @@ export class ActionBarComponent extends BaseComponent implements OnInit {
     @Output()
     backButtonClick: EventEmitter<void> = new EventEmitter();
 
-    constructor(_cd: ChangeDetectorRef) {
+    navigationArrow$: Observable<string>;
+
+    constructor(private _rtlService: RtlService, _cd: ChangeDetectorRef) {
         super(_cd);
     }
 
-    ngOnInit() {
-        if (this.contentDensity === 'cozy') {
-            this.compact = false;
-        }
+    ngOnInit(): void {
+
+        this.navigationArrow$ = this._rtlService.rtl.pipe(

```

Comment:
I am thinking if its not more effective to use angular/cdk the https://material.angular.io/cdk/bidi/api, I am using it in select, but this is just a question. You can definitely stick with this RTL service. I just dont like the observable approach. You either load this page within RTL lang or dont.